src/any_llm/providers/gemini/interactions.py

```
@@ -0,0 +1,343 @@
+        "model",
+        "input",
+        "instructions",
+        "max_output_tokens",
+        "stream",
+        "store",
+        "metadata",
+        "previous_response_id",
+    }
+    unsupported = params.model_dump(exclude_none=True).keys() - supported
+    if unsupported:
+        raise UnsupportedParameterError(sorted(unsupported)[0], provider_name)
```

Ruff (0.16.3)

[error] Prefer `min` over `sorted()` to compute the minimum value in a sequence

Replace with `min`

(FURB192)